README.md

```diff
@@ -78,8 +132,78 @@ log:
   notify_errors: admin@example.com
 ```
 
-**NOTE**: If you are using a config file, all other flags and environment
-variables will be ignored.
+## Docker
+[![Docker Build Status](https://img.shields.io/docker/build/nols1000/padlock-cloud.svg?style=flat-square)](https://hub.docker.com/r/nols1000/padlock-cloud/)
```

Comment:
```suggestion
[![Docker Build Status](https://img.shields.io/docker/build/padlock/padlock-cloud.svg?style=flat-square)](https://hub.docker.com/r/padlock/padlock-cloud/)
```

Reply:
I created a docker hub organisation for hosting the official repo. The automated build is already set up - should be good to go once this PR is merged.